src/assets/_project/_blocks/layout/header/site-search.html

```diff
@@ -1,14 +1,14 @@
-<form action="https://pan.search.qld.gov.au/s/search.html" id="qg-search-form" role="search" class="collapse">
-        <label for="qg-search-query" class="qg-visually-hidden">Search Queensland Government</label>
-        <div class="input-group">
-            <input type="text" name="query" id="qg-search-query" class="form-control" placeholder="Search website" tabindex="0" />
-            <span class="input-group-btn">
-                <button type="submit" id="feature-search-submit" class="btn btn-primary" title="Search site"><span class="fa fa-search" aria-hidden="true"></span><span class="qg-visually-hidden">search</span></button>
-            </span>
-        </div>
-        <!--parameters to display results on search page-->
-        <input type="hidden" name="num_ranks" value="10">
-        <input type="hidden" name="tiers" value="off">
-        <input type="hidden" name="collection" value="qld-gov">
-        <input type="hidden" name="profile" value="qld">
+<form action="https://pan.search.qld.gov.au/s/search.html" id="qg-search-form" role="search" class="collapse">  
+  <label for="qg-search-query" class="qg-visually-hidden">Search Queensland Government</label>
+  <div class="input-group">
+    <input type="text" name="query" id="qg-search-query" class="form-control" placeholder="Search website" tabindex="0" />
+    <span class="input-group-btn">
+      <button type="submit" id="feature-search-submit" class="btn btn-primary" title="Search site"><span class="fa fa-search" aria-hidden="true"></span><span class="qg-visually-hidden">search</span></button>
```

Comment:
Hope this button is inline element. Else its not semantic to place block element inside inline element (span)